lib/HLSL/DxilPreparePasses.cpp

```diff
@@ -640,6 +640,83 @@ class DxilFinalizeModule : public ModulePass {
     }
   }
 
+  void convertIsInf(Module &M, CallInst *CI) {
```

Comment:
```suggestion
  static void ConvertIsInf(Module &M, CallInst *CI) {
```

(and similar suggestions for the other new functions added that don't really need to be member functions - I get you're following the existing pattern in this file, so will leave it as a judgement call if you take this suggestion or now)